modules/extensions/src/api/data.ts

```diff
@@ -8,8 +8,8 @@ import { extensionPort } from "../util/comlink";
 /**
  * Fetches the current user via graphql
  */
-export async function currentUser(args: CurrentUserDataInclusion) {
-  return await extensionPort.currentUser(args);
+export async function currentUser(args?: CurrentUserDataInclusion) {
+  return await extensionPort.currentUser(args || {});
```

Comment:
i like this syntax, it brings the default initialization value to the parameter definition

```suggestion
export async function currentUser(args: CurrentUserDataInclusion = {}) {
  return await extensionPort.currentUser(args);
```